client/beefy/src/lib.rs

```diff
@@ -126,8 +126,6 @@ where
 	pub signed_commitment_sender: BeefySignedCommitmentSender<B>,
 	/// BEEFY best block sender
 	pub beefy_best_block_sender: BeefyBestBlockSender<B>,
-	/// Minimal delta between blocks, BEEFY should vote for
-	pub min_block_delta: u32,
```

Comment:
Let's not remove this. You can still use it on top of your changes as described in [beefy.md #L224](https://github.com/paritytech/grandpa-bridge-gadget/blob/ff8c141244cf62c61a4a1f646423712b0722820d/docs/beefy.md?plain=1#L224).